server/src-lib/Hasura/Server/Compression.hs

```diff
@@ -7,22 +7,23 @@ where
 
 import           Hasura.Prelude
 
-import           Hasura.Server.Utils       (brHeader, gzipHeader)
+-- import           Hasura.Server.Utils       (brHeader, gzipHeader)
```

Comment:
I think these comments should be removed rather than just commented out—we can always get the deleted code back by reverting this commit.